ci/install.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Setup configuration
+cp ci/testsuite.cfg.$DB testsuite.cfg
+sudo cp ci/haas.cfg.$DB /etc/haas.cfg
+sudo chown travis:travis /etc/haas.cfg
+
+# Database Setup
+if [ $DB = postgres ]; then
+    psql --version
+    psql -c 'CREATE DATABASE haas_tests;' -U postgres
+    psql -c 'CREATE DATABASE haas;' -U postgres
+elif [ $DB = sqlite ]; then
+    # Workaround to create an empty sqlite db file
```

Comment:
When removing it `haas-admin db create` throws a few warnings but everything still seems to work fine, having it there removed them.